src/Licenses/LicensesClient.php

```diff
@@ -203,6 +209,40 @@ public function getConfigsRaw(string $reference): string
         return $this->get();
     }
 
+    /**
+     * @param string $reference
+     *
+     * @return string
+     *
+     * @throws GuzzleException
+     * @throws NotFoundException
+     * @throws PublicApiClientException
+     */
+    public function getPredictionRaw(string $reference): string
```

Comment:
I think this method should be named `getPredictionsDailyRaw()`
You should also add an optional `$parameters` path to add some more parameters in the url (see other examples)
```suggestion
    public function getPredictionsDailyRaw(string $reference, array $parameters = []): string
```

Reply:
> this was added then removed we had a start date parameter in path parameter and an end date once i did the first demo for nader , luc and william and vincent we decided to remove it since XBE already gets the daily consumption and the predictions only start  from the last day of consumption used by XBE to the end of the billing period .